lighthouse/hooks/cherrypick_test_data.py

```diff
@@ -65,3 +59,15 @@ def inserted_cherrypick_test_data_hook(runs: List[Dict[str, Any]]) -> None:
                 HTTPStatus.INTERNAL_SERVER_ERROR,
             )
         )
+
+
+def get_httperror_message(error):
+    try:
+        response_json = error.response.json()
+    except Exception:
+        return str(error)
+
+    if isinstance(response_json, dict) and "errors" in response_json:
```

Comment:
I think it's more pythonic to "Ask For Forgiveness" than checking types and if keys are in dicts. Sometimes tricky to find the balance: https://devblogs.microsoft.com/python/idiomatic-python-eafp-versus-lbyl/

Reply:
Fair enough.  It's a bit too much of a culture change for my brain to accept, but I can see the benefits.  I also hadn't fully resolved in my head that because the action is still to `return str(error)`, the exception caused by trying to read a key that doesn't exist can be caught by the existing `except `block.

Reply:
All good, it is not something that comes naturally to me either, I always want to check things first!